services/zigbee/__main__.py

```
@@ -0,0 +1,169 @@
+        })
+
+    app.register_blueprint(
+        create_blueprint(get_zigpy_controller=lambda: _zigpy_controller)
+    )
+    return app
+
+
+def _run_api_server(app: Flask) -> None:
+    log = logging.getLogger(__name__)
+    try:
+        app.run(host="0.0.0.0", port=PORT, debug=False, use_reloader=False)
```

Ruff (0.15.13)

[error] Possible binding to all interfaces

(S104)